Author: Sean Finney <seanius@debian.org>
Description: Fix FTBFS: "gconf/gconf-client.h: No such file or directory"
 It seems that @GNOME_WINDOW_SETTINGS_LIBS@ used to implicitly include
 gconf-related compiler flags, but not always.  This should explicitly
 include them.
Thanks: Sebastian Harl <tokkee@debian.org>
Origin: vendor, debian
Bug-Debian: http://bugs.debian.org/562027
Bug-Debian: http://bugs.debian.org/562438

Author: Jasmine Hassan <jasmine.aura@gmail.com>
Description: migrated for MATE desktop
Last-Updated: 2012-10-09

--- a/gtk/gnome/Makefile.am
+++ b/gtk/gnome/Makefile.am
@@ -2,7 +2,8 @@ moduledir = $(DESTDIR)$(windowsettingsli
 
 if USE_MATE
 libcompiz_la_LDFLAGS = -export-dynamic -avoid-version -rpath $(moduledir)
-libcompiz_la_LIBADD  = @MATE_WINDOW_SETTINGS_LIBS@
+libcompiz_la_LIBADD  = @MATE_WINDOW_SETTINGS_LIBS@ @MATECONF_LIBS@
+libcompiz_la_CFLAGS  = @MATECONF_CFLAGS@
 libcompiz_la_SOURCES = 		\
 	compiz-window-manager.c	\
 	compiz-window-manager.h
